chokidar/chokidar-tests.ts

```diff
@@ -36,6 +41,7 @@ watcher.unwatch('new-file*');
 watcher.close();
 
 // One-liner
-require('chokidar').watch('.', {ignored: /[\/\\]\./}).on('all', function(event:string, path:string) {
-    console.log(event, path);
+chokidar.watch('.', {ignored: /[\/\\]\./}).on('all', (event, path) => {
```

Comment:
`require('chokidar')` isn't type-checked. Had to add an `"all"` overload to fix this test.